RocketChat/Rocket.Chat.ReactNative
# feat: align mobile videoConf settings with web settings

## Proposed changes


Align mobile videoConf settings with web settings.
https://github.com/RocketChat/Rocket.Chat/blob/8ac075833579f0e1429763f1d3ae6b1abc383229/apps/meteor/app/videobridge/client/tabBar.tsx

## Issue(s)	


## How to test or reproduce

Test all call possibilities:
- Start a call through the header.
- Start a call through the actions screen.
- Start a call through RoomInfo.

Check if the rules are the same as on the web, in channels, teams, direct, archived channels or channels that are prohibited from sending messages. 

## Screenshots

## Types of changes




## Checklist



## Further comments


Jira: [VC-130]


[VC-130]: https://rocketchat.atlassian.net/browse/VC-130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
